app/Modules/Position/Position.php

```diff
@@ -29,13 +31,18 @@ class Position extends Model
         self::TYPE_TERMINAL => ['TMA', 'Terminal'],
         self::TYPE_FSS => ['FSS', 'Flight Service Station'],
     ];
-    protected $fillable = ['callsign', 'name', 'frequency', 'type'];
+    protected $fillable = ['callsign', 'name', 'frequency', 'type', 'airfield'];
```

Comment:
best practice to have this property called `airfield_icao`, such that the relationship can be `airfield()`?

Reply:
yeah, I was in two minds about this, but you've convinced me

Reply:
all sorted